midi_app_controller/state/state_manager.py

```diff
@@ -101,40 +101,48 @@ def get_available_midi_out(self) -> List[str]:
         """Returns names of all MIDI output ports."""
         return self._midi_out.get_ports()
 
-    def select_binds(self, name: str) -> None:
+    def select_binds(self, name: Optional[str]) -> None:
         """Updates currently selected binds.
 
         Does not have any immediate effect except updating the value and
-        finding path to the config file.
+        finding path to the config file if the name is not None.
```

Comment:
```suggestion
        finding the path to the config file if the name is not None.
```
Not sure, but sounds better to me.